### DISCUSSION TITLE
v 1.9.0 RC1

### DISCUSSION THREAD

**Discussion by pubpub-zz** [pubpub-zz/ppInk#290] [Announcements]
ppInk : version with support to OBS studio and REST API
ppInkS : version without support to OBS studio and REST API
Major changes
FIX: many fix (cf change log)
ENH: accelerate snapshots
ENH: Add capability to edit filling of  drawings
ENH: confirm width=0 in pen edit and makes invisible nul width
ENH: reactivate Default_Arrow direction
FIX: crash because of comma as decimal separator
ENH: Add Measure during drawing and scale redefinition
ENH : add page of extra pens
ENH: allow to define hotspot for  cliparts
NEW: Add Pages feature
ENH: fading in dashed/straight line
ENH: add ppInkS version
ENH : add defaults to update config.ini and english to update translations
ENH : engage line when pressing shift or ctrl in hand drawing
ENH : add option to keep toolbar unfolded in pointer mode

<hr /><em>This discussion was created from the release <a href='https://github.com/pubpub-zz/ppInk/releases/tag/'>v 1.9.0 RC1</a>.</em>

**Comment by ghost** [comment]
![Ekran görüntüsü 2024-06-05 041609](https://github.com/pubpub-zz/ppInk/assets/171755386/480fec1b-d739-4bc4-86ee-727daa1360b9)
i have problem how can i solver

**Reply by pubpub-zz** [reply]
can you please create a new issue to track it, and provide more details on what you were doing, and also attach the ini files

**Reply by wrathofflan** [reply]
I don't read Turkish (?) but I think I got a similar message. I've been using ppInk every day for years and after I did the Window 11 update today I got this 
![ppInk won't open anymore](https://github.com/user-attachments/assets/becf2fdb-b161-4c2c-be7a-f84b5e4658cc)

I deleted everything and downloaded it anew. Got this message a second time

**Comment by ghost** [comment]
hello i have the same issue until 1.6 rc1 im using 1.6 rc1 and still pretty
good. im using ryzen apu ryzen 7 5825  and i am the guy who do the all the
update guy so maybe because of this and at this problem there is missing
problems says that E disk i dont have any E Disk maybe because of this but
im during this issues after trying other versions then 1.6 and older
versions running pretty good but i just delete it de other versions and
thank u for the program. bcs i hate epicpen

pubpub-zz ***@***.***>, 5 Haz 2024 Çar, 23:23 tarihinde şunu
yazdı:

**Reply by pubpub-zz** [reply]
@lifelover4 can you open a new issue attaching the log when the issue is occurring and confirm or retest with default in files
Also more details on what you were doing would be appreciated

**Comment by Rafatrl** [comment]
O problema persiste desde 2024 sem nenhuma solução.
O Fluxo:
Baixo qualquer zip desses:


<img width="221" height="130" alt="image" src="https://github.com/user-attachments/assets/2d2caf95-d70a-4bbc-a07c-b918d0ed403c" />



Executo como admin o arquivo `ppInk.exe` e aparece essa mesma tela que todos estao reportando:
<img width="449" height="464" alt="image" src="https://github.com/user-attachments/assets/7d52c9c2-d163-4e58-9de1-0ae5eebc3521" />

**Comment by 52HZAlice** [comment]
关于：未处理的异常
，版本1.9.0.20296，构建于2024年4月30日11:16。哎呀，pplnk程序崩溃了！

“[zhaoxx1]”先生的解决方案有效，卸载卸载系统更新 KB5065426 以解决
具体步骤：设置-Windows更新-更新历史记录-卸载更新

"Mr. [zhaoxx1]'s solution is effective; uninstall system update KB5065426 to resolve the issue. Specific steps: Settings - Windows Update - Update History - Uninstall Updates.

<img width="387" height="401" alt="v3CxJ3IQmu" src="https://github.com/user-attachments/assets/3b74b326-f53a-4e73-80bc-e5991ed2c612" />

**Comment by pujokoe** [comment]
The solution is just install 7zip and extract using it, not using windows default unzip app. It works in 25H2. I think this version is the most less buggy than the later update version.
